activerecord/lib/active_record/inheritance.rb

```diff
@@ -37,6 +37,7 @@ module Inheritance
 
     included do
       # Determines whether to store the full constant name including namespace when using STI.
+      # This is true, by default.
```

Comment:
Do we need to say it's true by default when two lines later we have the `= true`? 😬